### ISSUE TITLE
Issue When Opening Newer RAOBs

### ISSUE THREAD

**Issue by jpan84** [sharppy/SHARPpy#117]
I take FSL format files from esrl.noaa.gov/raobs/ and convert to SHARPpy's tabular form using my own program.  Older data have no issues.  OTOH, often, when I try to open RAOB data from after 2011 or so, I get the following error:

"index -1 is out of bounds for axis 0 with size 0"

[87111112.txt](https://github.com/sharppy/SHARPpy/files/1009302/87111112.txt)

[12103003.txt](https://github.com/sharppy/SHARPpy/files/1009296/12103003.txt)

**Comment by georgephillips1** [comment]
Don't know if this is the problem, but the surface level at 961 mb shown in the FSL output is not included in your text file in the 2012 sounding data.

**Comment by jpan84** [comment]
@georgephillips1 you're right.  I deleted that because it caused height out of order error.  Should I rearrange it instead of deleting?

**Comment by georgephillips1** [comment]
@jpan84 that's what I did.  I had to do a sort to get the right format.  I was working with FSL output file from your 2012 case, and there are other issues with a direct conversion that you will likely find.  One is that there are two cases with the same pressure level/height duplicated.  SHARPpy doesn't like that.  The other is that the FSL output has 360 degrees for a due north wind, whereas SHARPpy expects 0 degrees.  I opened a ticket on the 2nd issue and maybe should on the first as well.  I can code workarounds for both in my conversion program, but that doesn't help the next person writing a conversion program.
